src/components/date/date.spec.js

```diff
@@ -465,7 +465,7 @@ describe("Date", () => {
       onFocusFn = jest.fn();
     });
 
-    describe.each([
+    describe.skip.each([
```

Comment:
I think you need to remove the `.skip` from this test. Don't want this getting merged to `master`